repose-aggregator/artifacts/docker/src/docker/centos/Dockerfile

```diff
@@ -14,16 +14,24 @@ RUN yum update -q -y && yum install -q -y wget java-1.8.0-openjdk-headless
 RUN wget --quiet -O /etc/yum.repos.d/openrepose.repo http://repo.openrepose.org/el/openrepose.repo
 RUN yum update -q -y && yum install -q -y repose-valve-$REPOSE_VERSION repose-filter-bundle-$REPOSE_VERSION repose-extensions-filter-bundle-$REPOSE_VERSION repose-experimental-filter-bundle-$REPOSE_VERSION
 
-# Expose /etc/repose as a volume for mounting.
-WORKDIR /etc/repose
-VOLUME /etc/repose
+ENV APP_ROOT=/etc/repose
+ENV APP_VARS=/var/repose
+ENV APP_LOGS=/var/log/repose
```

Comment:
I suspect you had keep this reference around because phone-home-messages are still being written to a file in this directory, and log4j2 seems to touch (i.e., create) the repose log file even though nothing is appending to is.

So in my eyes, we can either:
1. Keep this around to handle the auxiliary phone home logging and log4j2 file management.
1. Remove this and remove all file appenders from the log4j2 configuration in our Docker image.

Reply:
Yep, as far as the `/var/log/repose` reference goes.
I'm good commenting out both file appenders and removing this all together.
I'll do that.